golang/helpers/apiHelper.go

```diff
@@ -0,0 +1,65 @@
+package helpers
```

Comment:
reusable code should go in a `pkg` folder/package - more of an idiomatic golang structure

Reply:
I don't think the file names need to have "helper" in them - `api` / `identity` should suffice if they're in a `pkg` folder.